src/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.cs

```diff
@@ -114,68 +114,201 @@ public enum TokenAccessLevels
+        AccountEnterpriseAdminsSid = 48,
+        /// <summary>Indicates a SID that matches the policy administrators group.</summary>
+        AccountPolicyAdminsSid = 49,
+        /// <summary>Indicates a SID that matches the RAS and IAS server account.</summary>
+        AccountRasAndIasServersSid = 50,
+        /// <summary>Indicates a SID present when the Microsoft NTLM authentication package authenticated the client.</summary>
         NtlmAuthenticationSid = 51,
-        NullSid = 0,
-        OtherOrganizationSid = 55,
-        ProxySid = 14,
-        RemoteLogonIdSid = 20,
-        RestrictedCodeSid = 18,
+        /// <summary>Indicates a SID present when the Microsoft Digest authentication package authenticated the client.</summary>
+        DigestAuthenticationSid = 52,
+        /// <summary>Indicates a SID present when the Secure Channel (SSL/TLS) authentication package authenticated the client.</summary>
         SChannelAuthenticationSid = 53,
-        SelfSid = 16,
-        ServiceSid = 12,
-        TerminalServerSid = 19,
+        /// <summary>Indicates a SID present when the user authenticated from within the forest or across a trust that does not have the selective authentication option enabled. If this SID is present, then <see cref="WinOtherOrganizationSid"/> cannot be present.</summary>
         ThisOrganizationSid = 54,
+        /// <summary>Indicates a SID present when the user authenticated across a forest with the selective authentication option enabled. If this SID is present, then <see cref="WinThisOrganizationSid"/> cannot be present.</summary>
+        OtherOrganizationSid = 55,
+        /// <summary>Indicates a SID that allows a user to create incoming forest trusts. It is added to the token of users who are a member of the Incoming Forest Trust Builders built-in group in the root domain of the forest.</summary>
+        BuiltinIncomingForestTrustBuildersSid = 56,
+        /// <summary>Indicates a SID that matches the performance monitor user group.</summary>
+        BuiltinPerformanceMonitoringUsersSid = 57,
+        /// <summary>Indicates a SID that matches the performance log user group.</summary>
+        BuiltinPerformanceLoggingUsersSid = 58,
+        /// <summary>Indicates a SID that matches the Windows Authorization Access group.</summary>
+        BuiltinAuthorizationAccessSid = 59,
+        /// <summary>Indicates a SID is present in a server that can issue terminal server licenses.</summary>
         WinBuiltinTerminalServerLicenseServersSid = 60,
-        WorldSid = 1,
+        [System.ObsoleteAttribute("This member has been depcreated and is only maintained for backwards compatability. WellKnownSidType values greater than MaxDefined may be defined in future releases.")]
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        MaxDefined = WinBuiltinTerminalServerLicenseServersSid,
+        #if NETSTANDARD20
```

Comment:
Ah misunderstanding.... https://github.com/dotnet/corefx/issues/14636#issuecomment-270160041 suggested the `#ifdef`. But let's remove it and see what happens...